src/main/java/edu/cornell/kfs/sys/service/impl/ISOFIPSConversionServiceImpl.java

```diff
@@ -0,0 +1,109 @@
+ * used by the rest of the KFS system.
+ *
+ *    *******NOTE*******
+ * The FIPS country codes are stored in KFS table SH_CNTRY_T which is manually
+ * maintained by the Country maintenace document.
+ * Due to the manual nature of maintaining those FIPS country code values,
+ * there are some INACTIVE ISO country codes also existing in that FIPS table.
+ *    *******NOTE*******
+ *
+ * When mapping between the two different standards, in some cases a country
+ * will have the same country code in both the FIPS and ISO standard.
+ *
+ * In other cases, the same country will be represented by different country
+ * codes in the FIPS and ISO standards.
+ *
+ * There are also cases where multiple distinct FIPS country codes map to the
+ * same single ISO country code.
+ *
+ * Specific examples for each of the three data conditions stated above are listed below.
+ *
+ *   ============                             ===========
+ *   FIPS-Country                             ISO Country
+ *   ============                             ===========
+ *   US = United States                       US = United States of America (the)
+ *
+ *   JA = Japan                               JP = Japan
+ *   AJ = Azerbaijan                          AZ = Azerbaijan
+ *   No FIPS code exists                      AS = American Samoa
+ *   GB = Gabon                               GA = Gabon
+ *   UK = United Kingdom                      GB = United Kingdom of Great Britain and Northern Ireland (the)
+ *
+ *   BQ = NA VASSA ISLAND                     UM = United States Minor Outlying Islands (the)
+ *   DQ = JARVIS ISALND                       UM = United States Minor Outlying Islands (the)
+ *   FQ = BAKER ISLAND                        UM = United States Minor Outlying Islands (the)
+ *   HQ = HOWLAND ISLAND                      UM = United States Minor Outlying Islands (the)
+ *   JQ = JOHNSTON ATOLL                      UM = United States Minor Outlying Islands (the)
+ *   KQ = KINGMAN REEF                        UM = United States Minor Outlying Islands (the)
+ *   LQ = PALMYRA ATOLL                       UM = United States Minor Outlying Islands (the)
+ *   MQ = MIDWAY ISLANDS                      UM = United States Minor Outlying Islands (the)
+ *   WQ = WAKE ISLAND                         UM = United States Minor Outlying Islands (the)
+ *
+ *   AS = AUSTRALIA                           AU = Australia
+ *   AT = ASHMORE AND CARTIER ISLANDS         AU = Australia
+ *   CR = CORAL SEA ISLANDS TERRITORY         AU = Australia
+ *
+ *****************************************************************************/
+
+public class ISOFIPSConversionServiceImpl implements ISOFIPSConversionService {
+    
+    private static final Logger LOG = LogManager.getLogger(ISOFIPSConversionServiceImpl.class);
+    
+    private ISOFIPSCountryMapDao isoFipsCountryMapDao;
+    
+    /*
+     * ISO-to-FIPS mappings are many-ISO-to-one-FIPS.
```

Comment:
The "many" and "one" sides in this comment seem to contradict the direction stated in the larger Javadoc comment. Please update or remove this comment accordingly.

Reply:
I will remove.